plugins/Toolbox/src/Toolbox.py

```diff
@@ -61,17 +59,15 @@ def __init__(self, application: CuraApplication) -> None:
         self._server_response_data = {
             "authors":              [],
             "packages":             [],
-            "updates":              [],
-            "subscribed_packages":  [],
+            "updates":              []
```

Comment:
I think it's better to leave 'end of list' comma's in place, so git-blame works (more smoothly). It's not a big deal though, so we can leave it.

Reply:
Agree, will do that in the future